bblfsh-dashboard/templates/install-bblfshd-drivers-configmap.yaml

```diff
@@ -8,6 +8,6 @@ metadata:
 data:
   install-bblfshd-drivers.sh: |
     {{- range $language, $driver := .Values.server.drivers.languages }}
-    bblfshctl driver install {{ $language }} {{ $driver.repository }}:{{ $driver.tag }}
+    bblfshctl driver install {{ $language }} {{ $driver.repository }}:{{ $driver.tag }} --ctl-endpoint=tcp --ctl-address=localhost:9433
```

Comment:
why do we need this? Is it because when we activate ctl endpoint we cannot communicate through the unix socket anymore?